docs/circuit_cutting/tutorials/gate_cutting_to_reduce_circuit_depth.ipynb

```diff
@@ -0,0 +1,423 @@
+      "text/plain": [
+       "<Figure size 1834.66x361.2 with 1 Axes>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from circuit_knitting_toolbox.circuit_cutting.qpd import decompose_qpd_instructions\n",
+    "\n",
+    "# Set some arbitrary bases to which the QPD gates should decompose\n",
+    "for idx in cut_indices:\n",
+    "    qpd_circuit[idx].operation.basis_id = 2\n",
```

Comment:
Awesome, you got `basis_id = 2` to work :+1:

Reply:
Yes, I was calling `QuantumCircuit.decompose()`, with no arguments, so it was sending `QPDMeasure` gates to the transpiler, which it doesnt know what to do with.

I had to use `decompose_qpd_instructions` to properly get everything to decompose down. I think it's cool to flash little pieces of the underlying functionality here and there in the tutorials and how-tos without distracting too much from the purpose of the doc